aexpect/client.py

```diff
@@ -54,6 +54,8 @@
 
 _THREAD_KILL_REQUESTED = threading.Event()
 
+logging = log.getLogger(__name__)
```

Comment:
Most of the places updated recently on the Avocado side use `LOG` (uppercase) as the variable name. There are still old entries in the utilities using lowercase, so it is not uniform yet. It would be good to make this change uniform within and across the projects.

Anyway, this is just a comment, not a blocker.

Reply:
Yes, I am also wondering about the `s/logging/log/` change. In any case I have mostly seen projects use `log` and the capitalized `LOG` seems to be an Avocado thing. Granted, this is an Avocado subproject but it could also be generalized beyond this scope I think. In any case not important, I am fine by either choice.

Reply:
> In any case not important, I am fine by either choice.

Same here.